source/reference/verification.txt

```diff
@@ -16,114 +16,96 @@ Verify Data Transfer
    :depth: 2
    :class: singlecol
 
-When :ref:`mongosync <c2c-mongosync>` has fully committed, verify the 
-successful transfer of your data before you switch your application to using 
-the destination cluster. You can verify your data transfer using document 
-counts, hash comparison, document comparison, or the Migration Verifier.
+When :program:`mongosync` has fully committed, verify the
+successful transfer of your data before you switch your
+application to using the destination cluster.
 
-Use Cases
----------
+.. versionadded:: 1.9
 
-You should verify your data after every sync. This is important in
-cases where you plan to move your application load from the source to the
-destination cluster.
-
-Verification methods:
-
-- :ref:`c2c-verify-with-doc-counts`
-
-- :ref:`c2c-verify-with-hash-comp`
-
-- :ref:`c2c-verify-with-doc-comp`
-
-- :ref:`c2c-verify-with-verifier`
-
-The specific method you use to verify your data depends on your application
-workload and the complexity of the data.
+   ``mongosync`` provides an embedded verifier that can perform
+   a series of verification checks on the source and destination
+   cluster to ensure a successful sync.
 
 .. _c2c-verify-method:
 
 Tasks
--------
-
-.. _c2c-verify-with-doc-counts:
-
-Document Counts
-~~~~~~~~~~~~~~~
-
-The most basic method of verification is to compare the number of documents
-in each synced collection on the source cluster to the number on the
-destination cluster.
-
-This method only verifies a successful sync when run against clusters with
-insert-only workloads.
-
-For more information, see :ref:`c2c-verify-doc-counts`.
-
-.. _c2c-verify-with-hash-comp:
-
-Hash Comparison
-~~~~~~~~~~~~~~~
-
-You can verify sync by comparing MD5 hashes of collections synced from the
-source cluster to the destination cluster.
-
-While hash comparison ensures that the destination cluster has received all
-changes from the source, the :dbcommand:`dbHash` command locks the cluster,
-preventing additional writes until it completes.
+-----
+
+You should verify your data after every sync. This is important
+in cases where you plan to move your application load from the
+source to the destination cluster.
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Verification Method
```

Comment:
@mvankeulen94 @jtcovan, I believe all of these are possible for the alternative verification methods, except maybe the Migration Verifier. Does that sound right?

Reply:
Migration verifier can verify all of those data points.

The other verification methods listed are very primitive, like "write a script for document comparison". I don't know if it's helpful to enumerate the cases this can support (all cases, in theory). 
I think it would be good to add a something on this page similar to the document comparison for `indexes, collection , metadata, views`. Users would need to write a custom script or manually inspect the data for these as well.

Reply:
@mvankeulen94 Addendum: I can't reply to this [comment](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/339#discussion_r1823212895) for some reason, but I added a "contact support" message to the section just above.

Reply:
Can we add a point about how users should verify indexes, collection metadata, and views? All of these methods refer to document data right now. This would be by writing a custom script or manually inspecting these components.

Reply:
Fixed